### ISSUE TITLE
Support "Convert from nuget"

### ISSUE THREAD

**Issue by forki** [fsprojects/Paket.VisualStudio#25]
http://fsprojects.github.io/Paket/paket-convert-from-nuget.html

how could we integrate this into the plugin. At this point we don't have paket.dependencies yet.

// @theimowski

**Comment by theimowski** [comment]
Could probably start with invoking the static member https://github.com/fsprojects/Paket/blob/master/src/Paket.Core/PublicAPI.fs#L59

and see what explodes?

**Comment by forki** [comment]
Yep I know about that part. I'm more interested in discussing the UX for that when I open a VS solution that uses nuget. Where/how would we start that process?

**Comment by theimowski** [comment]
I see, how about these areas:
![image](https://cloud.githubusercontent.com/assets/5909015/7607089/f8fda89a-f95e-11e4-8234-a329f8db5fb5.png)

**Comment by forki** [comment]
yep that sounds good.
@hmemcpy now how can we create such a menu? ;-)

**Comment by hmemcpy** [comment]
In general, all menu creation in VS is a matter of adding an entry in the .vsct file, and then finding the correct "parent" magic string to insert it to (using `CommandPlacement`).
There used to be some utilities to dump all commands ids in VS, but I don't know if it works (it was VSCT PowerToy, I think)
Anyway, all those magic strings are listed in [`vsshlids.h`](https://code.google.com/p/costura/source/browse/Tools/vs2010.sdk/inc/vsshlids.h)...

**Comment by hmemcpy** [comment]
Wait, that's the long, and unneceserrily complicated explaination :)
To put the menu under Tools, all you need to do is to set its command placement into `IDM_VS_MENU_TOOLS`
Here is the full list: https://msdn.microsoft.com/en-us/library/cc826118(v=vs.140).aspx

**Comment by forki** [comment]
I tried to folow https://msdn.microsoft.com/en-us/library/bb165473(v=vs.140).aspx and https://msdn.microsoft.com/en-us/library/bb165739(v=vs.140).aspx but I don't get it to compile.

**Comment by hmemcpy** [comment]
Where exactly do you want to place this item?

**Comment by forki** [comment]
TOOLS/ Paket Dependency Manager/Convert from NuGet  !?

**Comment by hmemcpy** [comment]
So you want to add a top-level menu under Tools? That shouldn't be a problem... hold on

**Comment by efrainsteinbach** [comment]
How about right-clicking on "packages.config" file --> convert to "paket.references" ? (which is what the command is actually doing, no?)
